### DISCUSSION TITLE
Adding Kanto ORA Data

### DISCUSSION THREAD

**Discussion by sk9592** [pierreaubert/spinorama#267] [General]
Hi Pierre, 

I love your website and have found it incredibly useful. I was wondering whether you would be able to add the Kanto ORA data from Audioholics to your site:

https://www.audioholics.com/computer-speaker-reviews/kanto-living-ora/#toc-h2-1

Thank you!

**Comment by pierreaubert** [comment]
Hello,

yes I can but no guarantee on the timeline. You can also scan the graphs
yourself to speed up things. The tutorial to do so is here.
Even better, can you convince whoever has the original data to share them?
Having the raw data would minimise errors.

Pierre

On Wed, Apr 10, 2024 at 1:45 AM sk9592 ***@***.***> wrote:

> Hi Pierre,
>
> I love your website and have found it incredibly useful. I was wondering
> whether you would be able to add the Kanto ORA data from Audioholics to
> your site:
>
>
> https://www.audioholics.com/computer-speaker-reviews/kanto-living-ora/#toc-h2-1
>
> Thank you!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pierreaubert/spinorama/discussions/267>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANBKH52F4A7MIR2OW3JJY3Y4R4TJAVCNFSM6AAAAABF7OLN3SVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGQ4DIOJQGA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
--
PfA

**Reply by sk9592** [reply]
The original data was created by James Larson of Audioholics. I don't know how receptive they are about sharing raw data. If I can find his contact info, I will try asking though.

Thanks!
